src/Native/Runtime/RHCodeMan.cpp

```diff
@@ -189,6 +189,31 @@ void ReportRegisterSet(UInt8 regSet, REGDISPLAY * pContext, GCEnumContext * hCal
     if (regSet & CSR_MASK_X28) { ReportObject(hCallback, GetRegObjectAddr<CSR_NUM_X28>(pContext), 0); }
 }
 
+#elif defined(_TARGET_WASM_)
```

Comment:
Nothing in this file is applicable to Portable flavor of the runtime. It would be better to exclude it from the portable build instead of adding ifdefs to it. (If some of the methods are require to make things compile, add NYI stubs to portable.cpp.)

Reply:
Done

Reply:
On further inspection, removing this file from portable is a mess since EEMethodInfo and EECodeManager are in here and there are many dependencies on those in the Module class. Is it ok to just keep this file in the portable runtime? If so, this file depends on gcdump, so I'd need to keep that as well.

Reply:
I have submitted PR to compile this for ProjectN only: https://github.com/dotnet/corert/pull/4924

Reply:
Thanks! I've removed changes here and in gcdump.